src/main/java/org/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -270,8 +270,8 @@ private void writeText(String text, int startPos, int endPos) {
 
     private void writeStringLabel(String text, int startPos, int endPos,
                                   boolean first, boolean last) {
-        putIn((first ? "" : " # ") + text.substring(startPos, endPos)
-                + (last ? "" : " # "));
+        putIn((first ? "{" : " # ") + text.substring(startPos, endPos)
```

Comment:
Can you please explain the reason for this change. Thanks!

Reply:
This method were create only for "#%s#" forma I guess. The output should be "{%s}" to have this result for month
```
month = {mon}
```
not this
```
month = mon
``` 
[check this on #3239](https://github.com/JabRef/jabref/issues/3239#issuecomment-353082567)

Reply:
Mhh, I'm pretty sure that `month = jan` is the correct form and not `month = {jan}`. Maybe @koppor can clarify this.

Reply:
As we know that bibTeX forma should be 
```
key = {value}
```
u can add a debug instruction here [LatexFieldFormatter.java#L166](https://github.com/DevSiroukane/jabref/blob/67387113513ce9529e08b62ffc430d13e24f0b4b/src/main/java/org/jabref/logic/bibtex/LatexFieldFormatter.java#L166)
```
System.out.println(stringBuilder);
```
and see the out put after each change in any field. You will find that only month output is wrong if we miss the {} that I add on 
```
LatexFieldFormatter.java#writeStringLabel(String text, int startPos, int endPos, boolean first, boolean last)
```

Reply:
@DevSiroukane You are confusing it here. If it's  `#month#` it should be displayed without braces in the source. So the entry editor part was already correct. 
Only when it's  `month` -> `{month}`

Reply:
@Siedlerchr are you sure that month should not be between {} on source tab?

because I try to use generate entry using DOI (ex: [10.1109/TSG.2014.2346511](https://ieeexplore.ieee.org/document/6884808/)) and the out put is this on source tab:
```
@Article{Rahbari-Asr2014,
  author    = {Navid Rahbari-Asr and Unnati Ojha and Ziang Zhang and Mo-Yuen Chow},
  title     = {Incremental Welfare Consensus Algorithm for Cooperative Distributed Generation/Demand Response in Smart Grid},
  journal   = {{IEEE} Transactions on Smart Grid},
  year      = {2014},
  volume    = {5},
  number    = {6},
  pages     = {2836--2845},
  month     = {nov},
  doi       = {10.1109/tsg.2014.2346511},
  publisher = {Institute of Electrical and Electronics Engineers ({IEEE})},
}
```

Reply:
@Siedlerchr I think I should let source tab month appear as before
```
month = mon
```
I will remove the change as you said @tobiasdiez